stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/ChannelListItemViewHolder.java

```diff
@@ -177,7 +179,8 @@ protected void configLastMessage(Channel channel) {
     }
 
     protected void configLastMessageDate(Channel channel) {
-        tv_date.setText(DateFormatter.formatAsTimeOrDate(channel.getLastMessageAt()));
+        String date = DateFormatterKt.formatDate(defaultDateFormatter, channel.getLastMessageAt());
```

Comment:
If we make this `formatDate` public, we should rename the generated containing class for nicer Java API. Depends on the previous comment about it being public / internal though.

Reply:
Let me add it